### PR TITLE
new cards 💳 

### LINKED ISSUE #62: Info
Advertising ID

### PR DESCRIPTION
**Pull request by emmajam**
Added:
- diners club 
- maestro 
- visa electron

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -62,7 +63,10 @@ const BACKSPACE_KEY_CODE = 8;
 const CARD_TYPES = {
   mastercard: 'MASTERCARD',
   visa: 'VISA',
-  amex: 'AMERICAN_EXPRESS'
+  amex: 'AMERICAN_EXPRESS',
+  dinersclub: 'DINERS_CLUB',
+  maestro: 'MAESTRO',
```
Comment (jxom) [inline src/index.js]: Maybe add Discover as well?

File: src/index.js [hunk 2]
```diff
@@ -116,6 +121,21 @@ class CreditCardInput extends Component {
     }
   }
 
+  validateExpirationDate (date) {
```
Comment (jxom) [inline src/index.js:124]: Looks like the expiration date fix snuck in somehow!

**Review by jxom — APPROVED** [review]
Looks good! I'll worry about adding in card images later. :)

File: src/index.js (hunk 2 above)
Comment (emmajam) [reply src/index.js:124]: NOooo

**Comment by jxom** [comment]
Closing this for now until there is demand for support of these types of cards.

**Pull request closed without merging**